packages/utils/src/lib/collect/implementation/utils.spec.ts

```diff
@@ -60,3 +60,49 @@ describe('sumRefs', () => {
     expect(sumRefs(refs)).toBe(11);
   });
 });
+
+describe('formatBytes', () => {
+  it('should log file sizes in Bytes`', async () => {
+    expect(formatBytes(1000)).toBe('1000 B');
+  });
+
+  it('should log file sizes in KB`', async () => {
+    expect(formatBytes(10000)).toBe('9.77 kB');
+  });
+
+  it('should log file sizes in MB`', async () => {
+    expect(formatBytes(10000000)).toBe('9.54 MB');
+  });
+
+  it('should log file sizes in bytes`', async () => {
+    expect(formatBytes(10000000000)).toBe('9.31 GB');
+  });
+
+  it('should log file sizes in TB`', async () => {
+    expect(formatBytes(10000000000000)).toBe('9.09 TB');
+  });
+
+  it('should log file sizes in PB`', async () => {
+    expect(formatBytes(10000000000000000)).toBe('8.88 PB');
+  });
+
+  it('should log file sizes in EB`', async () => {
+    expect(formatBytes(10000000000000000000)).toBe('8.67 EB');
+  });
+
+  it('should log file sizes in ZB`', async () => {
+    expect(formatBytes(10000000000000000000000)).toBe('8.47 ZB');
+  });
+
+  it('should log file sizes in YB`', async () => {
+    expect(formatBytes(10000000000000000000000000)).toBe('8.27 YB');
+  });
+
+  it('should log file sizes correctly with correct decimal`', async () => {
+    expect(formatBytes(10000, 1)).toBe('9.8 kB');
+  });
+
+  it('should log file sizes of 0 if no size is given`', async () => {
+    expect(formatBytes(0)).toBe('0 B');
+  });
```

Comment:
Is the expected behavior that it logs a file with 0 bites ?

Reply:
It gets a number passed for display logic. So I was refrained to throw here.